examples/angularjs_require/js/controllers/todo.js

```diff
@@ -52,6 +52,8 @@ define(['app', 'services/todoStorage'], function (app) {
 
 			$scope.editTodo = function (todo) {
 				$scope.editedTodo = todo;
+				// Clone the original todo to restore it on demand.
+				$scope.originalTodo = angular.extend({}, todo);
```

Comment:
@passy All other angular examples do it this way so shall I fix it up in all of them?